Assets/HoloToolkit/Input/Scripts/GazeStabilizer.cs

```diff
@@ -136,15 +136,17 @@ private void UpdateInstability(out float positionInstability, out float directio
                 return;
             }
 
-            mostRecentSample = stabilitySamples.ElementAt(stabilitySamples.Count - 1);
+            mostRecentSample = stabilitySamples[(stabilitySamples.Count - 1)];
 
             // All but most recent.
             for (int i = 0; i < stabilitySamples.Count - 1; ++i)
```

Comment:
Alternatively you could go back to the Queue and iterate over it here instead of over the count as this loop doesn't seem to really require random element access.